src/main/java/com/mentorship/vineservice/dto/PaginationDto.java

```diff
@@ -0,0 +1,15 @@
+package com.mentorship.vineservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class PaginationDto {
```

Comment:
IMHO this dto is redundant too, we can just pass page and size as separate params